doc/nrf/gsg_guides/nrf7002_gs.rst

```diff
@@ -286,28 +286,20 @@ This allows to build and run secure-only applications on the nRF5340 SoC.
 Building |NCS| applications with Arm TrustZone
 ==============================================
 
-Applications on nRF5340 can use Cortex-M Security Extensions (CMSE) and separate firmware for the application core between Secure Processing Environment (SPE) and Non-Secure Processing Environment (NSPE).
-You can build SPE using either |NCS| or `Trusted Firmware M`_ (TF-M).
-You must always build NSPE using |NCS|.
+Applications on nRF5340 can use the M33 TrustZone for :ref:`security by separation <ug_tfm_security_by_separation>`.
```

Comment:
```suggestion
Applications on an nRF5340 device can use the M33 TrustZone for :ref:`security by separation <ug_tfm_security_by_separation>`.
```

Reply:
This is talking about the nRF5340 chip on the nRF7002 DK. There is no "nRF5340 device" in this context. Edit not applied.